app/[sheetId]/List.tsx

```diff
@@ -59,15 +59,15 @@ const PrimaryBlock = ({
   const values = attributes
     ?.map((attr: AttributeItem) => ({
       ...attr,
-      itemValue: getAttributeValue(item[camelCase(attr.title)]),
+      itemValue: getAttributeValue(item[camelCase(attr.columnName)]),
```

Comment:
you should keep it title here. we transform it when we do parseMeta. I mean you don't have to change because title should be there, columnName will be there too though. The build is currently failing because you need to change this in other places, I suggest not to change it at all, keep it title no need we rename it when we do this in reslitData:
```  title: property.columnName,```

Reply:
Ok, but it's confusing to me that we use "title" in the code for both a property and a type.